index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css"> 
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Chivo+Mono:wght@100&family=Noto+Sans+JP&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="header">
+    <header>My Logo</header>
+    <div class="menu-img"><img src="images/Union.svg"></div>
+    </div>
+    <section id="first-sec">
```

Comment:
- [ ] It is a core requirement that the content within this section is aligned with `flexbox`. Please ensure that you have used flexbox.